kwalify/standard/v1.0.0.yaml

```diff
@@ -0,0 +1,12 @@
+type: map
+mapping:
+  =:
+    type: map
+    mapping:
+      name:
+        type: str
+      family:
+        type: str
+
```

Comment:
Let's also include a `description` map of type `str` to allow for a short description of the control per #60